onnxruntime/test/onnx/main.cc

```diff
@@ -680,19 +680,6 @@ select from 'TF8', 'TF16', 'UINT8', 'FLOAT', 'ITENSOR'. \n)");
     {"test_scatternd_add", "Opset 16 not supported yet."},
     {"test_scatternd_multiply", "Opset 16 not supported yet."},
     {"test_scatter_elements_with_duplicate_indices", "Opset 16 not supported yet."},
-    {"resize_downsample_scales_cubic_antialias", "resize kernel needs update for opset18."},
```

Comment:
You might also want to update https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/test/testdata/onnx_backend_test_series_filters.jsonc .

Reply:
Please also remove the exclusions from `\csharp\test\Microsoft.ML.OnnxRuntime.Tests.NetCoreApp\InferenceTest.netcore.cs` given those tests now pass with your changes.

Reply:
Skipped or disabled tests from the two files don't have attribute of anti-aliasing. Not sure this PR can make it passed. Will have a try.